data/frm/Crecy_1997_xPonthus/stemma.gv

```diff
@@ -0,0 +1,23 @@
+digraph {
+
+x [color="grey"];
+x -> P1; 
+x -> P2; 
+x -> P3; 
+x -> P4; 
+
+x-> 1;
+1 [label="", color="grey"];
```

Comment:
Should this `1` be called `x¹` ?

Reply:
The letter x1 indicates a groupe of manuscripts (b), composed by G and P5 (not represented in the stemma).

Reply:
I have thus assumed that they have a common ancestor, without name (1).